test/benchmark/sqlite/ext2_benchmarks/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+/benchmark/bin/sqlite-speedtest1 /ext2/test.db
```

Comment:
The default size for the SQLite benchmark is 100 which is too small. Could you add `--size 1000` to the script? Thanks

Reply:
Get it!